src/modules/site-v2/templates/releases/download_tab_strain_v2_issues.html

```diff
@@ -1,4 +1,12 @@
 
+{% set ns = namespace(hasIssue=false) %}

```

Comment:
prefer capitalized booleans (eg: `False`)

Reply:
Just pushed changes to capitalize.